doc/03_reference/verification.rst

```diff
@@ -223,6 +223,138 @@ You can add any compile/runtime options in `dv/uvm/core_ibex/yaml/simulator.yaml
+| Review        | V3_CHECKLIST_SCOPED                 | Complete   |                                                                                                                                                                      |
++---------------+-------------------------------------+------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+**PMP Testing Note**: A large number of iterations of ``pmp_full_random_test`` are required to meet coverage goals and timed out tests must be included in the coverage collection.
+This is because of the large cross bins for PMP that aim to explore the full space of possible behaviour.
+The current strategy of random generation is very inefficient at exploring this space.
+It is also complex to write a randomly generated test that can deal with all possible scenarios without hitting a double faulting or time out scenarios (e.g. consider a random configuration that gives you no executable regions).
+Co-simulation checking is enabled when this test is run (as it is for all block level verification tests) so would detect any incorrect behaviour.
+From investigation we are confident the time-outs seen are simply badly performing tests (e.g. very slowly working its way through an instruction block with no execute permissions by attempting to execute one instruction, faulting, trying the next and getting the same result over and over).
+For future work we will explore more efficient strategies for exploring this space as well as employing formal methods to achieve full verification closure.
+
+Ibex V2S Checklist
+""""""""""""""""""
+
++---------------+--------------------------+------------+--------------------------------------------------------------------------------------------------------------------------------------+
+| Type          | Item                     | Resolution | Notes                                                                                                                                |
++===============+==========================+============+======================================================================================================================================+
+| Documentation | SEC_CM_TESTPLAN_COMPLETE | Complete   | Security counter measure to test mapping can be found below                                                                          |
++---------------+--------------------------+------------+--------------------------------------------------------------------------------------------------------------------------------------+
+| Tests         | FPV_SEC_CM_VERIFIED      | Complete   | See the `OpenTitan FPV Results Dashboard <https://reports.opentitan.org/hw/top_earlgrey/formal/sec_cm/summary/latest/report.html>`_. |
```

Comment:
(Should soon show up on dashboard, now that https://github.com/lowRISC/opentitan/pull/16156 is merged.)